### ISSUE TITLE
K8S adapter cleanup

### ISSUE THREAD

**Issue by f1ames** [scramjetorg/transform-hub#268]
This is a follow-up of https://github.com/scramjetorg/transform-hub/pull/247. We have agreed to left some things unpolished there, especially in context of #267.

So what has left:

* [ ] Unpack only package.json (ideally to stdout) in KubernetesSequenceAdapter 
* [ ] Store only compressed package on disk, such as `.scramjet_sequences/<seq-id>.tar.gz`
* [ ] Figure out a way to get rid of redundant scripts `unpack.sh` and `run.sh` from Runner docker image
* [ ] Support [topic feature tests](https://github.com/scramjetorg/transform-hub/blob/bae0c49bc9783bf8ce41aaccbea0369358c2a971/bdd/features/e2e/E2E-013-topic.feature)
* [ ] Investigate objLogger not printing in KubernetesInstanceAdapter
* [ ] Extract objLogger improvements to a separate PR
* [ ] Make yarn clean use `git clean -xdf` or also remove packed sequences
* [ ] Support getting runner PID in BDD tests (see [here](https://github.com/scramjetorg/transform-hub/blob/09567dff17f6f1d53cb3cd3990332df1c4f463ef/bdd/step-definitions/e2e/host-steps.ts#L337-L340))
* [ ] Support checking if runner was closed successfully in BDD tests (see [here](https://github.com/scramjetorg/transform-hub/blob/09567dff17f6f1d53cb3cd3990332df1c4f463ef/bdd/step-definitions/e2e/host-steps.ts#L360-L363))
* [ ] Get rid of all "no-docker" occurences
* [ ] Runner pod `imagePullPolicy: "Always"` only for development (see https://github.com/scramjetorg/transform-hub/pull/247#discussion_r8118832100

Probably it will be most efficient to cover it together or after #267.
